pkg/cephmgr/mon/log.go

```diff
@@ -0,0 +1,20 @@
+/*
+Copyright 2016 The Rook Authors. All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+	http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+package mon
+
+import "github.com/coreos/pkg/capnslog"
+
+var logger = capnslog.NewPackageLogger("github.com/rook/rook", "cephmgr/mon")
```

Comment:
Fully qualified package names:
```
2016-11-15 09:16:50.222381 I | cephmgr/osd: Adding local path to local directory /var/lib/rook
```

Compared to unique (shortened) package name:
```
2016-11-15 09:16:50.222381 I | cephosd: Adding local path to local directory /var/lib/rook
```
I like the unique shortened approach better.